nvflare/app_opt/xgboost/histogram_based_v2/adaptor_executor.py

```diff
@@ -78,9 +113,11 @@ def handle_event(self, event_type: str, fl_ctx: FLContext):
                 )
                 return
 
+            self.sender = self.get_sender(fl_ctx)
+
```

Comment:
If self.sender is None (system_panic already called), then should return here.

Reply:
Done.